src/core/main.c

```diff
@@ -3223,7 +3224,8 @@ int main(int argc, char *argv[]) {
                 /* clear the kernel timestamp, because we are not PID 1 */
                 kernel_timestamp = DUAL_TIMESTAMP_NULL;
 
-                if (mac_init() < 0) {
+                r = mac_init();
+                if (r < 0) {
```

Comment:
please split this commit out, it has little to do with the user stuff and can be merged more quickly